src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -50,18 +51,23 @@ export class AutomatedChecksView extends React.Component<AutomatedChecksViewProp
         return (
             <div className={automatedChecksView}>
                 <TitleBar deps={this.props.deps} windowStateStoreData={this.props.windowStateStoreData}></TitleBar>
-                <div className={mainContentWrapper}>
-                    <CommandBar
-                        deps={this.props.deps}
-                        deviceStoreData={this.props.deviceStoreData}
-                        scanStoreData={this.props.scanStoreData}
-                    />
-                    <main>
-                        <HeaderSection />
-                        {this.renderScanning()}
-                        {this.renderDeviceDisconnected()}
-                        {this.renderResults()}
-                    </main>
+                <div className={resultsAndScreenshot}>
```

Comment:
I'm not sure about this name but I don't have a better one. I would like to have another opinion from @dbjorge and @lisli1

Reply:
I struggled with naming the similar concept in web when I redid that a month ago. The reason the div exists is in service of laying out the page into different panels, so maybe something like `automatedChecksPanelLayout`?